api/firmware/eth.go

```diff
@@ -141,6 +141,17 @@ func (device *Device) handleSignerNonceCommitment(response *messages.ETHResponse
 	return signature, nil
 }
 
+func ETHIdentifyCase(recipientAddress string) messages.ETHAddressCase {
+	switch {
+	case strings.ToUpper(recipientAddress) == recipientAddress:
+		return messages.ETHAddressCase_ETH_ADDRESS_CASE_UPPER
+	case strings.ToLower(recipientAddress) == recipientAddress:
+		return messages.ETHAddressCase_ETH_ADDRESS_CASE_LOWER
+	default:
+		return messages.ETHAddressCase_ETH_ADDRESS_CASE_MIXED
+	}
+}
+
 // ETHSign signs an ethereum transaction. It returns a 65 byte signature (R, S, and 1 byte recID).
```

Comment:
Add to the docstring what recipientAddressCase does and mention `ETHIdentifyCase` exists as a convenience function.